lib/clowder-common-ruby/schema.json

```diff
@@ -215,6 +215,10 @@
                 "rdsCa": {
                     "description": "Defines the CA used to access the database.",
                     "type": "string"
+                },
```

Comment:
I suggest make changes in sync_config.sh, rather than here. The change like:
```
diff --git a/sync_config.sh b/sync_config.sh
index 74ebcbd..8266708 100755
--- a/sync_config.sh
+++ b/sync_config.sh
@@ -1,4 +1,4 @@
 #!/bin/bash
 cd bin/
-wget https://raw.githubusercontent.com/RedHatInsights/clowder/master/controllers/cloud.redhat.com/config/schema.json -O schema.json
-./json_schema_ruby -o ../lib/clowder-common-ruby/types.rb schema.json
+wget https://raw.githubusercontent.com/RedHatInsights/clowder/master/controllers/cloud.redhat.com/config/schema.json -O ../lib/clowder-common-ruby/schema.json
+./json_schema_ruby -o ../lib/clowder-common-ruby/types.rb ../lib/clowder-common-ruby/schema.json
```

This will update both `types.rb` and `schema.json` files.